src/phoenix/config.py

```diff
@@ -21,6 +22,16 @@
 """
 The project name to use when logging traces and evals. defaults to 'default'.
 """
+ENV_SPAN_STORAGE_TYPE = "__DANGEROUS__PHOENIX_SPAN_STORAGE_TYPE"
+"""
+**EXPERIMENTAL**
+The type of span storage to use.
+"""
+ENV_SPAN_STORAGE_PATH = "__DANGEROUS__PHOENIX_SPAN_STORAGE_PATH"
```

Comment:
curious why you made this configurable?

Reply:
no particular reason other than completeness, i can also take it out